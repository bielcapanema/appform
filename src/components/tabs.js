function tabs() {
  return `<div class="wrapper">
    <h2>App Form</h2>
    <h6>Utilizando VanillaJS</h6>
    <nav class="tabs">
      <div class="selector"></div>
      <a onClick="window.goTo('/cadastro')" class="${window.location.pathname === '/cadastro' ? 'active' : ''}">Cadastro</a>
      <a onClick="window.goTo('/lista')"  class="${window.location.pathname === '/lista' ? 'active' : ''}">Lista</a>
    </nav>
  </div>`
}

export default tabs