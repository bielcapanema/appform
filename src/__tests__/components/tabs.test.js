import tabs from '../../components/tabs'

test('Tabs render', () => {
  expect(tabs()).toMatchSnapshot()
})