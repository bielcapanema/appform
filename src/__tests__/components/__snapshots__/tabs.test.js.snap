// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tabs render 1`] = `
"<div class=\\"wrapper\\">
    <h2>App Form</h2>
    <h6>Utilizando VanillaJS</h6>
    <nav class=\\"tabs\\">
      <div class=\\"selector\\"></div>
      <a onClick=\\"window.goTo('/cadastro')\\" class=\\"\\">Cadastro</a>
      <a onClick=\\"window.goTo('/lista')\\"  class=\\"\\">Lista</a>
    </nav>
  </div>"
`;
